packages/docs/src/vite-plugin-synergy-styles/toCem.ts

```diff
@@ -33,7 +41,14 @@ const getTypeForTag = (tag: Tag) => {
  * @param tag The tag to check
  * @returns True if the tag is allowed to be included
  */
-const tagIsAllowedToBeIncluded = (tag: Tag): boolean => tag.tag === 'variant';
+const tagIsAllowedToBeIncluded = (tag: Tag): boolean => ['variant', 'boolean'].includes(tag.tag);
+
+/**
+ * Check if a tag is a boolean type
+ * @param tag The tag to check
+ * @returns True if the tag is a boolean type
+ */
+const tagIsBooleanType = (tag: Tag): boolean => tag.tag === 'boolean' || !tag.type || tag.type.trim().length === 0 || tag.type === 'true';
```

Comment:
Why is a documentation value of `false` not also a boolean?
Just from the docs, I think both of these lines should be a boolean:

```css
@boolean { true } syn-table-cell Applies the table cell styling
@boolean { false } syn-table-cell Applies the table cell styling
```

Reply:
if you would write the documentation like this, it would be handled correctly. Because then the `tag.tag === 'boolean'` would come to action. But if someone would for some reason has the idea to do this (use `variant`instead of `boolean`):

```js
@variant { true } syn-table-cell Applies the table cell styling
```

then it would also use the boolean type. But I'm not sure if we need this at all..
Maybe we should just remove this, because if someone wants to do a boolean class he should just use `@boolean` and not `@variant`. 
Or what do you think? 
If we let it like this, maybe I should just add `|| tag.type === 'false` so this is also possible: 
```js
@variant { false } syn-table-cell Applies the table cell styling
```

Reply:
Ah, I get it. So currently, you could use `@variant` as a boolean trigger, too? Not sure where we want to go with this. On the one hand, it would distinguish booleans and variants when using the first approach. On the other hand, when allowing the second one, we could skip `@boolean` completely. However in this case, you would have problems with comments like this:

```css
@variant { false } syn-table-cell Clearly a boolean
@variant { true } syn-table-cell Clearly a boolean
@variant { false | true } syn-table-cell Clearly a boolean
@variant { false | true | blue } syn-table-cell This is not a boolean anymore?
```

Reply:
In the case of the last two examples it would not get triggered. Because the type would be either `type = 'false | true'` or for the other case `type = 'false | true | blue`

Reply:
Fine for me, lets stick with that.